client/dashboard/app/query-client.ts

```diff
@@ -22,10 +22,15 @@ const [ , persistPromise ] = persistQueryClient( {
 	buster: '2', // Bump when query data shape changes.
 	maxAge,
 	dehydrateOptions: {
-		shouldDehydrateQuery: ( { queryKey } ) => {
+		shouldDehydrateQuery: ( { queryKey, meta } ) => {
 			if ( TWO_STEP_QUERY_KEY === queryKey ) {
```

Comment:
Why not remove it?

Reply:
I think @ellatrix meant why not remove this `if ( TWO_STEP_QUERY_KEY === 'queryKey' )` line, CMIIW.

Reply:
I see. Yes, this can be removed.